onebot/plugins/psa.py

```diff
@@ -29,5 +29,8 @@ def psa(self, mask, target, args):
 
             %%psa <message>...
         """
+        msg = ' '.join(args['<message>'] or [])
```

Comment:
If you're going to do `or []` I'd rather you return an error message: sending an empty PSA seems silly.